src/TransformationValidator.php

```diff
@@ -58,9 +60,15 @@ private function validateBucketRows(): void
         }
 
         array_walk($this->config['rows'], function ($v): void {
-            if (!isset($v['configuration']['id'])) {
+            $missingConfig = [];
+            foreach (self::REQUIRED_ROWS_CONFIG as $item) {
+                if (!isset($v['configuration'][$item])) {
+                    $missingConfig[] = $item;
+                }
+            }
+            if ($missingConfig) {
                 throw new CheckConfigException(sprintf(
-                    'Transformation "%s" is not configured.',
+                    'Transformation "%s" is empty. Please add querries or mapping to continue with migration.',
```

Comment:
@ondrajodas Drobnost, je tu typo v `querries`. ;)

Ale mozno by to slo formulovat inak. Nie ze "pridaj si queries alebo mapping ak chces migrovat", ale napr. "Transformacia XYZ je prazdna, nie je potrebne ju migrovat".

Reply:
jj za typo díky 👍 

k obsahu message.... takto to přišlo od Tomis, asi by to šlo napsat více způsoby

Reply:
Ok, tak potom len to typo :shrug: